$NetBSD: patch-ad,v 1.2 2012/04/16 04:33:30 dholland Exp $

- use standard headers
- don't declare own errno
- don't declare own strpbrk() either

--- refile.c.orig	1997-07-04 17:30:10.000000000 +0000
+++ refile.c
@@ -42,11 +42,8 @@ static char XRNrcsid[] = "$Id: refile.c,
 #else
 #include <sys/file.h>
 #endif
-#if defined(_XOPEN_SOURCE)
 #include <stdlib.h>
-#else
-    char *getenv();
-#endif
+#include <string.h>
 #include <errno.h>
 
 #include "xrn.h"
@@ -57,8 +54,6 @@ static char XRNrcsid[] = "$Id: refile.c,
 #include "mesg_strings.h"
 #include "refile.h"
 
-extern char *strpbrk();
-
 #ifndef S_ISDIR
 #define S_ISDIR(mode) (((mode) & S_IFMT) == S_IFDIR)
 #endif
@@ -125,7 +120,6 @@ int MHrefile(folder, artfile)
     char newfolders[512];
     char msg[512];
     struct stat st;
-    extern int errno;
 
    
     p = getenv("HOME");
@@ -202,7 +196,6 @@ int RMAILrefile(fullpath, folder, artfil
 {
     char msg[512];
     struct stat st;
-    extern int errno;
     FILE * fp;
     int artfd, rv, n;
 
